jsonforms-editor/src/palette-panel/components/SchemaTree.tsx

```diff
@@ -73,20 +73,19 @@ const getChildrenToRender = (schemaElement: SchemaElement) => {
 
 export const SchemaTreeView: React.FC<{
   schema: SchemaElement | undefined;
-}> = ({ schema }) => {
-  if (schema === undefined) {
-    return <NoSchema />;
-  }
-  return (
-    <>
-      <Typography variant='h6' color='inherit' noWrap>
-        Controls
-      </Typography>
+}> = ({ schema }) => (
+  <>
+    <Typography variant='h6' color='inherit' noWrap>
+      Controls
+    </Typography>
+    {schema ? (
```

Comment:
A schema consisting of `false` would also be valid, so we should check for undefined here I guess ;)